next.config.js

```diff
@@ -5,27 +5,57 @@ const withCss = require('@zeit/next-css');
 const withSass = require('@zeit/next-sass');
 const withFonts = require('next-fonts');
 
+const SentryWebpackPlugin = require('@sentry/webpack-plugin');
```

Comment:
Do not need the release integration. Remove the Webpack Plugin.

Reply:
Done